gramps/gen/lib/grampstype.py

```diff
@@ -24,12 +24,20 @@
 Base type for all gramps types.
 """
 
+
+# ---------------------------------------------------------------
+#
+# Python modules
+#
+# ---------------------------------------------------------------
+from __future__ import annotations
+from typing import Any
+
 # -------------------------------------------------------------------------
 #
 # Python modules
 #
 # -------------------------------------------------------------------------
```

Comment:
```suggestion

```
Remove duplicate comment block

Reply:
OK. I added this by mistake when I fixed the conflict.

Reply:
Done.